### PR TITLE
rc/1.118.0

### PR DESCRIPTION
**Pull request by samrichca**
# Description of changes

- #2457 
- #2449 
- #2456 
- #2440 
- chore: exclude graphql types from prettier

## Type of Change

- [ ] Bug
- [ ] Dependency updates
- [ ] Feature
- [ ] Refactoring
- [x] Release candidate
- [ ] Styling
- [ ] Testing
- [ ] Other (please describe)

### CODE REVIEW

File: components/pages/submission-system/program-submitted-data/SearchBar/FilterModal/index.tsx [hunk 1]
```diff
@@ -83,24 +91,32 @@ export default function FilterModal({
...
 
-      if (filteredTextAreaIDs.has(donorId)) {
-        filteredTextAreaIDs.delete(donorId);
-      } else if (filteredTextAreaIDs.has(result.submitterDonorId)) {
+      if (donorIdMatch || submitterIdMatch) {
+        filteredTextAreaIDs.delete(result.donorId);
         filteredTextAreaIDs.delete(result.submitterDonorId);
+
+        setMatchedIds([...matchedIds, result.donorId]);
       }
```
Comment (daniel-cy-lu) [inline components/pages/submission-system/program-submitted-data/SearchBar/FilterModal/index.tsx:120]: Changes in this useEffect() caused a bug in the unmatched count in filter modal. Unmatched number is always at least "1". I am looking at possible solution.

File: components/pages/submission-system/program-clinical-submission/Instruction/index.tsx [hunk 2]
```diff
@@ -171,7 +171,7 @@ const Instruction = ({
         </>,
         <>
           <Typography variant="data" component="span">
-            4. When your clinical data is valid and QC is complete, sign off your submission:
+            3. When your clinical data is valid and QC is complete, sign off your submission:
           </Typography>
```
Comment (daniel-cy-lu) [inline components/pages/submission-system/program-clinical-submission/Instruction/index.tsx:175]: Some editing of strings. Tested in DEV.

File: components/pages/submission-system/program-submitted-data/SearchBar/FilterModal/index.tsx (hunk 1 above)
Comment (samrichca) [reply components/pages/submission-system/program-submitted-data/SearchBar/FilterModal/index.tsx:120]: no worries, take your time

File: components/pages/submission-system/program-submitted-data/SearchBar/index.tsx [hunk 3]
```diff
@@ -134,31 +137,39 @@ export default function SearchBar({
       .slice(0, 20) || [];
 
   const titleText =
-    keyword && searchResultItems.length === 1
-      ? `${searchResultItems[0].resultId}`
+    currentDonors.length === 1
+      ? `DO${currentDonors[0]}`
+      : currentDonors.length > 1
+      ? `${currentDonors.length} Donors`
```
Comment (daniel-cy-lu) [inline components/pages/submission-system/program-submitted-data/SearchBar/index.tsx:143]: Thanks, Dan :) @demariadaniel

File: components/pages/submission-system/program-submitted-data/SearchBar/FilterModal/index.tsx (hunk 1 above)
Comment (demariadaniel) [reply components/pages/submission-system/program-submitted-data/SearchBar/FilterModal/index.tsx:120]: Fixed in https://github.com/icgc-argo/platform-ui/pull/2505

**Review by demariadaniel — APPROVED** [review]

**Comment by samrichca** [comment]
closing for new RC

**Pull request closed without merging**